XCode/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -55,4 +55,4 @@
     "version" : 1,
     "author" : "xcode"
   }
-}
\ No newline at end of file
+}
```

Comment:
Don't eat the newline at the end of files. (Luckily it looks like this is the only one.)

Reply:
That's actually adding a newline. Unless I'm misinterpreting what is wanted here?